packages/@aws-cdk/aws-elasticloadbalancing/test/integ.instanceTarget.elb.ts

```diff
@@ -14,16 +16,31 @@ const vpc = new ec2.Vpc(stack, 'VPC', {
 const instance = new ec2.Instance(stack, 'targetInstance', {
   vpc: vpc,
   instanceType: ec2.InstanceType.of( // t2.micro has free tier usage in aws
-    ec2.InstanceClass.T2,
+    ec2.InstanceClass.T3,
     ec2.InstanceSize.MICRO,
   ),
   machineImage: ec2.MachineImage.latestAmazonLinux({
     generation: ec2.AmazonLinuxGeneration.AMAZON_LINUX_2,
   }),
+  initOptions: {
+    timeout: cdk.Duration.minutes(30),
+  },
+  init: ec2.CloudFormationInit.fromElements(
+    ec2.InitService.systemdConfigFile('pythonweb', {
+      command: '/usr/bin/python3 -m http.server 8080',
+      cwd: '/var/www/html',
+    }),
+    ec2.InitService.enable('pythonweb', {
+      serviceManager: ec2.ServiceManager.SYSTEMD,
+    }),
+    ec2.InitFile.fromString('/var/www/html/index.html', 'Hello! You can see me!'),
+  ),
 });
+instance.role.addManagedPolicy(iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonSSMManagedInstanceCore'));
```

Comment:
Is the managed policy addition necessary or was this just for your own testing with session-manager or something?